src/collector/poller.rs

```diff
@@ -25,6 +26,17 @@ pub struct Poller<T: Middleware> {
     pool: PoolState<T>,
 }
 
+pub fn from_tick_weight(
+    tick_weight: &crate::time_range::TickWeight,
```

Comment:
You can add a `use` statement to import TickWeight so this is a bit cleaner.

Reply:
Fixed with this commit: https://github.com/PeggyJV/cellar_rebalancer_rs/pull/33/commits/ab548a39d77faf9ef220e4f47a54e2751c64cc04